charts/influxdb/README.md

```diff
@@ -215,8 +231,8 @@ spec:
             - |
               gsutil -m cp -r gs://<PATH TO BACKUP FOLDER>/* /backup
           volumeMounts:
-            - name: backup
-              mountPath: /backup
+            - name: {{ include "influxdb.fullname" . }}-backups
+              mountPath: {{ .Values.backup.directory | quote }}
```

Comment:
this is not a helm template, it is an example raw yaml of a restore job, please revert